src/serialbox-fortran/utils_ppser_kbuff.f90

```diff
@@ -113,6 +113,9 @@ SUBROUTINE finalize_kbuff()
   DO idx = 1, max_kbuff
     IF (kbuff(idx)%in_use) THEN
       WRITE(0,*) 'ERROR in utils_ppser_kbuff: finalize called before all buffers have been flushed'
+      WRITE(0,*) kbuff(idx)%savepoint_name
```

Comment:
Probably would make sense to add something like `WRITE(0,*) 'fieldname = ', TRIM(kbuff(idx)%fieldname)` for each line...